torch_xla/distributed/xla_backend.py

```diff
@@ -46,6 +46,13 @@ def __init__(self, prefix_store, rank, size, timeout):
   def getBackendName(self):
     return 'xla'
 
+  def _set_group_name(self, name: str) -> None:
+    self._group_name = name
+
+  @property
+  def group_name(self):
```

Comment:
Leave comment here explaining why we need to do this. It's very likely a PyTorch bug, since there's already a default implementation here: https://github.com/pytorch/pytorch/blob/7b1988f9222f3dec5cc2012afce84218199748ae/torch/csrc/distributed/c10d/ProcessGroup.cpp#L148-L152

We can file that bug separately, and clean this up later.